customerio/customerio-ios
# docs: [vision-on-sample-app][7/n] App Intro View

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #617
* #613
* #612
* #611
* __->__ #610
* #609
* #608
* #606
* #605
* #604

## Context

This is a PR in a series of PRs to build sample app that runs Identify and Track in VisionPro
The app is built to be an interactive walk through guidance for developers who are integrating Customer.io Swift SDK in their VisioPro apps.
For more context:

- See the project [one pager](https://www.notion.so/custio/Spacial-Identify-and-Tracking-6587d848bbde494095c0585237326ed3?pvs=4)
- See [Linear Issue](https://linear.app/customerio/issue/MBL-115/sample-app-for-visionpro)

## PR Summary
This PR introduces the first screen in the app when the app runs for the first time.
Intended to give the user an idea what to expect to see while using the app and quick call out on where they will find the actual `CustomerIO.[method]` calls if they all they care about is the code samples.

Beside the regular code review, I would really appreciate any suggestion to improve the visual/texutal content.


## Test plan
- Open `VisionOs.xcproject`
- Hit CMD+R to run the project and make sure you have VisionOS simulator installed and selected
- See the intro page similar to the one in the screenshot

<img width="1663" alt="AppIntro" src="https://github.com/customerio/customerio-ios/assets/142905621/d90b9f1e-0ae4-4711-9949-6264a17fbacb">
